sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/tests/OpenTelemetry.Exporter.AzureMonitor.IntegrationTests/OpenTelemetry.Exporter.AzureMonitor.Integration.Tests.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <!-- 
+  Integration tests evaluate an app's components on a broader level than unit tests. 
+  Unit tests are used to test isolated software components, such as individual class methods. 
+  Integration tests confirm that two or more app components work together to produce an expected result, 
+  possibly including every component required to fully process a request.
+  https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-3.1
+  -->
+  
+  <PropertyGroup>
+    <RequiredTargetFrameworks>netcoreapp2.1</RequiredTargetFrameworks>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- Suppressing the following warning because Microsoft.AspNetCore.Mvc.Testing is not available for net461. -->
+    <NoWarn>
+      NETSDK1005; <!-- NETSDK1005: Ensure that restore has run and that you have included 'net461' in the TargetFrameworks for your project. -->
+    </NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" VersionOverride="2.1.1" />
+
+    <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" VersionOverride="0.6.0-beta.1" />
```

Comment:
Consider extracting these versions to a common file before it becomes unmanageable.
Can be a separate PR so no need to get blocked here.